src/targets/services/categorize.js

```diff
@@ -37,16 +37,28 @@ export const categorize = async () => {
   } else {
     // Apply filters first
     let filteredOperations = filters.minOperationDate
-      ? (filteredOperations = operations.filter(
+      ? operations.filter(
           e =>
             new Date(e.date).valueOf() <
             new Date(filters.minOperationDate).valueOf()
-        ))
+        )
       : operations
 
     model = Model.fromDocs(filteredOperations)
   }
 
+  // Occurences per classes
+  let result = {}
+  for (let i in model.uniqueY) {
+    let sum = 0
+    for (let j = 0; j < model.occurences.length; j++) {
+      sum += model.occurences[j][i]
+    }
+    result[model.uniqueY[i]] = sum
+  }
+
+  console.log(result, model.occurences.length)
+
```

Comment:
If I understand correctly, `model.uniqueY` is the sum of all occurences for a particular class? But why do we need to do it now? It wasn't required before?

Reply:
`model.uniqueY` could be renamed to `model.uniqueCategories` as it's an array of all the categories and can be used to know the number of column.

Here it's just a debug computation I used to verify occurences. I will remove it